build/nativeaot-scenarios.yml

```diff
@@ -15,96 +15,111 @@ parameters:
   default:
 
   - displayName: Goldilocks Stage 1 (CoreCLR)
```

Comment:
So `Stage1` now has a different GC mode? And before that both `Stage1ServerGC` and `Stage1` where server GC?

Reply:
Just saw the changes in the csproj, it was actually Workstation implicitly. So fine.

Reply:
right - we are testing the 3 GC modes:

* ServerGC + DynamicAdapationMode (what we want to be the "default")
* ServerGC
* WorkstationGC (only run every 3 days)

It is a bit of a mess right now that we need to set the env var everywhere. But we can refactor that when the MSBuild property flows.